frappe/www/login.html

```diff
@@ -143,6 +143,35 @@ <h4>{{ _("Javascript is disabled on your browser") }}</h4>
 						<input type="email" id="signup_email" class="form-control"
 							placeholder="{{ _('jane@example.com') }}" required>
 					</div>
+					{% if show_country %}
+					<div class="form-group">
+						<div class="control-input-wrapper">
+							<div class="control-input flex align-center">
+								<select type="text" autocomplete="off" data-fieldtype="Select" data-fieldname="country"
```

Comment:
Let's use the `auto-complete` field. `Select field` for such a long list wouldn't be nice.
https://frappeframework.com/docs/v13/user/en/api/controls